Code/Framework/AzToolsFramework/AzToolsFramework/Metadata/UuidEntry.h

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/Serialization/SerializeContext.h>
+
+namespace AzToolsFramework
+{
+    //! Structure used to store UUID information for an asset in the metadata file
+    struct UuidEntry
```

Comment:
suggestion: call this `MetaUuid` instead of `UuidEntry` so it is more clear when devs search for the term "uuid" that this is for meta files specifically

Reply:
Good suggestion, I'll update that